integrations/operator/README.md

```diff
@@ -1,232 +1,106 @@
-| +---+----+------------------------+   |                |      |                     |
-|     |    |                            |                |      |                     |
-|     |    |                            |                |      |                     |
-|     |    |   operator                 |                |      |                     |
-| +---+----+----------------------------+--------+       |      |                     |
-| |   |    |                            |        |       |      |                     |
-| |  ++----+----+                 +-----v----+   |       |      |                     |
-| |  |  tBot    |-----------------> teleport <---+-------+------>                     |
-| |  +----------+ Get client      | operator |   |       |      |                     |
-| |              & renew certs    +----------+   |       |      |                     |
-| |                                              |       |      |                     |
-| |                                              |       |      |                     |
-| +----------------------------------------------+       |      +---------------------+
-|                                                        |
-|                                                        |
-|                                                        |
-+--------------------------------------------------------+
-```
-
-## Running
-
-### Requirementes
 
-#### K8S cluster
-If you don't have a cluster yet, you can start one by using the [minikube](https://minikube.sigs.k8s.io/docs/start/) tool.
-
-#### Operator's docker image
-You can obtain the docker image by pulling from `public.ecr.aws/gravitational/teleport`
-
-#### HELM chart from Teleport with the operator
-The [teleport-cluster Helm chart](../../examples/chart/teleport-cluster) supports deploying the operator alongside teleport.
-
-#### Other tools
-We also need the following tools: `helm`, `kubectl` and `docker`
-
-### Running the operator
-
-Install the helm chart:
-```bash
-# Run the command at the root of the teleport repo
-helm upgrade --install --create-namespace -n teleport-cluster \
-	--set clusterName=teleport-cluster.teleport-cluster.svc.cluster.local \
-	--set teleportVersionOverride="11.0.0-dev" \
-	--set operator.enabled=true \
-	teleport-cluster ./examples/chart/teleport-cluster
-
-kubectl config set-context --current --namespace teleport-cluster
+```mermaid
+graph TD
+    event([event])
+    eventType{Event type?}
+    event --> eventType
+
+    delete[Delete in Teleport]
+    eventType -- deletion --> delete
+    removeFinalizer[Remove finalizer]
+    delete -- success or 404 --> removeFinalizer
+    ending([end])
+    removeFinalizer --> ending
+
+
+    exists{Resource exists\nin Teleport ?}
+    addFinalizer[Add finalizer]
+    eventType -- create/update --> addFinalizer
+    addFinalizer --> exists
+
+    ownership{Origin label is\nkubernetes?}
+    exists -- yes --> ownership
```

Comment:
given the graph is beautiful, can you add negative paths to the graph and the retry scheduling -